scripts/tests/test_zkai_d64_rmsnorm_swiglu_statement_fixture.py

```
@@ -0,0 +1,188 @@
+        mutated = copy.deepcopy(statement)
+        mutated["model_id"] = "urn:zkai:ptvm:wrong-d64-block"
+        mutated["statement_commitment"] = FIXTURE._statement_commitment_from_payload(mutated)
+
+        with self.assertRaisesRegex(FIXTURE.StatementFixtureError, "model_id"):
+            FIXTURE.validate_statement(mutated)
+
+    def test_rows_for_tsv_rejects_stale_top_level_target(self) -> None:
+        payload = FIXTURE.build_fixture()
+        payload["target"]["width"] = 65
+
+        with self.assertRaisesRegex(FIXTURE.StatementFixtureError, "canonical target_spec"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)